File: cpp/arranjos/Vetor.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {

    int N;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> N;

    double vet[N];

    for (int i = 0; i < N; i++) {
        cout << "Digite um numero: ";
        cin >> vet[i];
    }

    cout << endl << "NUMEROS DIGITADOS:" << endl;

    cout << fixed << setprecision(1);

    for (int i = 0; i < N; i++) {
        cout << vet[i] << endl;
    }

    return 0;
}

File: java/curso-logica-java-nelio/Vetor.java
import java.util.Locale;
import java.util.Scanner;

public class Vetor {

	public static void main(String[] args) {
	
		Locale.setDefault(Locale.US);	
		Scanner leitor = new Scanner(System.in);
		
		System.out.print("Quantos numeros voce vai digitar? ");
		int N = leitor.nextInt();
		
		double[] vet = new double[N];
		
		for (int i = 0; i < N; i++) {
			System.out.print("Digite um numero: ");
			vet[i] = leitor.nextDouble();
		}
		
		System.out.println();
		System.out.println("NUMEROS DIGITADOS:");
		
		for (int i = 0; i < N; i++) {
			System.out.println(String.format("%.1f", vet[i]));
		}
		
		leitor.close();
	}
}